ludwig/schema/optimizers.py

```diff
@@ -32,7 +32,11 @@ def wrap(optimizer_config: BaseOptimizerConfig):
     return wrap
 
 
-@dataclass
+def get_optimizer_cls(name: str):
```

Comment:
Nit: add comment just bc this can be read the wrong way at a glance

Reply:
Good looks!